### DISCUSSION TITLE
Opening Screen

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#5186] [Help] [answered]
Somehow the screen that first opens when I click on "View Published app" has changed. I have 2 tables with budibase generated screen and it was opening with Table A's table list screen. It is now going to Table B's .id screen. I have no idea what is changed but it started after I had tried a "link screen" in Table A's settings choosing Table B's table list screen. I cleared the "Link screen" field. Still trying to open with Table B's :id.  Tried putting in the link to Table A's list screen. No luck. 

Is there any setting that tells the app which screen to first go to when you click on "View Published app"?

**Comment by melohagan** [comment, marked as answer]
If you set the route to '/' that indicates the home page, which should open on "View Published App"

![Screenshot 2022-03-31 at 09 16 52](https://user-images.githubusercontent.com/101575380/161009729-8cbfbd8d-f6e7-4289-a7e9-146c6e52cbfc.png)

**Reply by gpjorgenson** [reply]
Thanks. That does work although I also had to change the link in the layout to '/' as well.
